Packs/MicrosoftTeams/Scripts/MicrosoftTeamsAsk/MicrosoftTeamsAsk_test.py

```diff
@@ -3,9 +3,14 @@
 import demistomock as demisto
 import json
 import pytest
+from Packs.MicrosoftTeams.Integrations.MicrosoftTeams.MicrosoftTeams import MS_TEAMS_ASK_MESSAGE_KEYS
```

Comment:
I don't think we can do that :( Which is why we discussed two unit tests, here and in Teams, checking about the same thing.